src/subscription_manager/gui/widgets.py

```diff
@@ -971,14 +971,19 @@ def _update_cell_based_on_data(self, column, cell_renderer, tree_model, tree_ite
 
         if self.available_store_idx is not None:
             available = tree_model.get_value(tree_iter, self.available_store_idx)
-            if available and available != -1:
-                if self.quantity_increment_idx is not None:
-                    increment = tree_model.get_value(tree_iter, self.quantity_increment_idx)
-                else:
-                    increment = 1
 
-                cell_renderer.set_property("adjustment",
-                    ga_Gtk.Adjustment(lower=int(increment), upper=int(available), step_incr=int(increment)))
+            if self.quantity_increment_idx is not None:
+                increment = tree_model.get_value(tree_iter, self.quantity_increment_idx)
+            else:
+                increment = 1
+
+            if available:
+                if available != -1:
+                    cell_renderer.set_property("adjustment",
+                        ga_Gtk.Adjustment(lower=int(increment), upper=int(available), step_incr=int(increment)))
+                else:
+                    self.quantity_renderer.set_property("adjustment",
```

Comment:
`self.quantity_renderer` is `cell_renderer`. Please use `cell_renderer` in both places for consistency.